scene/gui/rich_text_label.cpp

```diff
@@ -4908,19 +4908,23 @@ TextServer::AutowrapMode RichTextLabel::get_autowrap_mode() const {
 	return autowrap_mode;
 }
 
-void RichTextLabel::set_percent_visible(float p_percent) {
+void RichTextLabel::set_percent_visible(const float p_percent) {
```

Comment:
We don't use `const` for trivial types. And here's it's inconsistent with `Label`, and inconsistent with `RichTextLabel`'s header (so I'm surprised it doesn't fail compiling).

Reply:
I don't even remember writing that, I'm a bit stumped it just works for some reason